gapic-generator/Gemfile.lock

```diff
@@ -134,7 +163,15 @@ GEM
     yard (0.9.37)
 
 PLATFORMS
-  x86_64-linux
+  aarch64-linux
+  aarch64-linux-gnu
+  aarch64-linux-musl
+  arm-linux-gnu
+  arm-linux-musl
+  arm64-darwin
+  x86_64-darwin
```

Comment:
Ugh. Can we remove the aarch64, arm, and darwin platforms?

Reply:
ack